.github/actions/setup-spark-builder/action.yaml

```diff
@@ -46,7 +42,6 @@ runs:
       run: |
         cd apache-spark
         git apply ../dev/diffs/${{inputs.spark-version}}.diff
-        ../mvnw -nsu -q versions:set-property -Dproperty=comet.version  -DnewVersion=${{inputs.comet-version}} -DgenerateBackupPoms=false
```

Comment:
This was hiding the fact that we had an incorrect version in the diff files. Now that this is removed, CI will fail if we do not have the correct version in the diff files, which is good since we want depend on these files having the correct version when running the Spark SQL tests locally.